transport/http/handler.go

```diff
@@ -82,6 +88,26 @@ func (h handler) callHandler(w http.ResponseWriter, req *http.Request) error {
 
 	ctx := context.Background()
 
+	// Extract opentracing etc baggage from headers
+	var spanCtx opentracing.SpanContext
+	if req.Header != nil {
+		carrier := opentracing.HTTPHeadersCarrier(req.Header)
+		spanCtx, _ = h.Tracer.Extract(opentracing.TextMap, carrier)
+	}
+
+	// Annotate the inbound context with a trace span
+	span := h.Tracer.StartSpan(
+		treq.Procedure,
+		opentracing.StartTime(now),
+		opentracing.Tags{
+			"caller":   treq.Caller,
```

Comment:
added the tag, retaining the yarpc tag names since they are always caller->service for both inbound and outbound, but peer.service varies by direction.